packages/storefront/tests/e2e/specs/accessibility.a11y.ts

```diff
@@ -71,19 +71,16 @@ describe('within main element', () => {
     const buttons = (
       await Promise.all([
         // page.$x("//button[text() = 'Vanilla JS']"),
-        page.$x("//button[text() = 'Angular']"),
-        page.$x("//button[text() = 'React']"),
+        page.waitForSelector("xpath///button[text() = 'Angular']"),
+        page.waitForSelector("xpath///button[text() = 'React']"),
```

Comment:
Isn't this something different now? In the past we got an array of `ElementHandle`s now we just get the first/single one?